[2025-03-24T12:34:46.967+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:34:47.183+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2022-10-02T06:00:00+00:00 [queued]>
[2025-03-24T12:34:47.274+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2022-10-02T06:00:00+00:00 [queued]>
[2025-03-24T12:34:47.274+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:34:47.367+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2022-10-02 06:00:00+00:00
[2025-03-24T12:34:47.388+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3036) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:34:47.416+0000] {standard_task_runner.py:72} INFO - Started process 3071 to run task
[2025-03-24T12:34:47.412+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2022-10-02T06:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmp7z_hab08']
[2025-03-24T12:34:47.434+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask ingest
[2025-03-24T12:34:47.745+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2022-10-02T06:00:00+00:00 [running]> on host 4791d3610749
[2025-03-24T12:34:48.547+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2022-10-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-02T06:00:00+00:00'
[2025-03-24T12:34:48.556+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:34:48.721+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:34:48.722+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-24T12:34:48.802+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:34:48.803+0000] {subprocess.py:106} INFO - ls
[2025-03-24T12:34:48.804+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:34:48.883+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:34:48.884+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2022-10-02T06:00:00+00:00, execution_date=20221002T060000, start_date=20250324T123447, end_date=20250324T123448
[2025-03-24T12:34:48.992+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:34:49.077+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:34:49.087+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
